_posts/2023-12-13-gift-guide.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: blog_post
+title: "Compiler's 2023 Holiday Gift Guide"
+subtitle: "Searching for that perfect gift for the public service enthusiast in your life? Compiler's got you covered! Here is this year's round-up of our favorite transit-related gift ideas.
+*(P.S. If you didn't make the holiday cut off this year, we think these make great gifts year-round!)*"
+description: ""
+author:
```

Comment:
If this was a collaboration, you could write `Compiler staff` for author. If not, put author name here.